### DISCUSSION TITLE
Questions about how to change background colors

### DISCUSSION THREAD

**Discussion by chenzhiqiang0518** [requarks/wiki#7341] [Help / Questions]
How can I change the blue background to light gray, and how can I change the text color to black? 

Thank you very much.
![image](https://github.com/user-attachments/assets/4649ca59-9345-4ab9-b15c-59c9e64c6105)

**Comment by heraldodinero** [comment]
I think your question is answered here:

https://github.com/requarks/wiki/discussions/7147
